src/signal/windows.rs

```diff
@@ -1,5 +1,7 @@
 //! Windows-specific types for signal handling.
 
+#[cfg(not(feature = "once_cell"))]
+use std::sync::OnceLock;
```

Comment:
We use `LazyLock` here, not `OnceLock`.

Reply:
Fixed by enable `lazy_cell` feature on nightly